vagrant/Vagrantfile

```diff
@@ -2,53 +2,73 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  if !Vagrant.has_plugin?("vagrant-hostsupdater")
-    raise "Please install vagrant-hostsupdater via `vagrant plugin install vagrant-hostsupdater`"
+  raise "Please install vagrant-hostsupdater via `vagrant plugin install vagrant-hostsupdater`" if !Vagrant.has_plugin?("vagrant-hostsupdater")
+
+  # additional plugin checks for base_image machine
+  if ARGV[1] == 'base_image'
+    raise "Please install vagrant-omnibus via `vagrant plugin install vagrant-omnibus --plugin-version=1.4.1`" if !Vagrant.has_plugin?("vagrant-omnibus")
+    raise "Please install vagrant-berkshelf via `vagrant plugin install vagrant-berkshelf`" if !Vagrant.has_plugin?("vagrant-berkshelf")
   end
 
   config.vm.synced_folder "../", "/baragon", type: "nfs"
-
-  # TODO: make my own box
-  config.vm.box = "singularity-develop-0.20.1"
-  config.vm.box_url = "https://vagrantcloud.com/tpetr/boxes/singularity-develop-0.20.1/versions/0.0.1/providers/virtualbox.box"
+  config.vm.box = "baragon-develop"
+  config.vm.box_url = "https://vagrantcloud.com/ssalinas/boxes/baragon-develop/versions/0.1.5/providers/virtualbox.box"
 
   config.vm.provider :virtualbox do |vb|
     vb.customize ["modifyvm", :id, "--memory", "1024"]
   end
 
-  if Vagrant.has_plugin?("vagrant-cachier")
-    config.cache.auto_detect = true
-  end
-
-  config.vm.define "service", autostart: true do |config|
-    config.vm.hostname = 'vagrant-baragon-service'
-    config.vm.network :private_network, ip: '192.168.33.20'
-
-    # This will enable the vagrants to connect to the zookeeper server on you local machine, comment out to only use the zookeeper server on the baragon-service machine
-    config.vm.network "forwarded_port", guest: 2181, host: 2181
+  config.cache.auto_detect = true if Vagrant.has_plugin?("vagrant-cachier")
```

Comment:
Need the check, otherwise if the plugin isn't install you get `Unknown configuration section 'cache'`